dask/dataframe/io/tests/test_parquet.py

```diff
@@ -4455,10 +4461,13 @@ def test_custom_filename_with_partition(tmpdir, engine):
 
 
 @PYARROW_MARK
-@pytest.mark.xfail(PANDAS_GT_200, reason="https://github.com/dask/dask/issues/9966")
 def test_roundtrip_partitioned_pyarrow_dataset(tmpdir, engine):
     # See: https://github.com/dask/dask/issues/8650
 
+    if engine == "fastparquet" and PANDAS_GT_200:
```

Comment:
What do you think about parametrizing instead?

```
@pytest.mark.parametrize("engine", [
	pytest.param("fastparquet", marks=pytest.mark.xfail(PANDAS_GT_200, reason="fastparquet reads as int64 while pyarrow does as int32")),
	"pyarrow"
])
```

Reply:
Sure sounds good.